checker/check-request-parameter-enum-value-updated.go

```diff
@@ -43,6 +43,17 @@ func RequestParameterEnumValueRemovedCheck(diffReport *diff.Diff, operationsSour
 							Source:      source,
 						})
 					}
+					for _, enumVal := range enumDiff.Added {
+						result = append(result, ApiChange{
+							Id:          "request-parameter-enum-value-added",
+							Level:       INFO,
+							Text:        fmt.Sprintf(config.i18n("request-parameter-enum-value-added"), enumVal, ColorizedValue(paramLocation), ColorizedValue(paramName)),
```

Comment:
I recommend to use ColorizedValue(enumVal), also in the other message, above

Reply:
Fixing for `request-parameter-enum-value-added` should be fine as we are introducing it now. 

But updating the other one, which already existed may mean we'll start adding '' on the enum value message, which is a change of output and could cause regressions to users if they have ignore-err with those values, right? What is the best option in that case? We can probably do the change but make sure we release a new minor version after this.

Reply:
Yes, it may break ignore files. Good point.
I would still recommend to make this correction, at the worst case, people will need to reconfigure their ignore files.
We will add a note about this in the release.

Regarding the version number:
Normally, we would need to increase the major version to signify incompatible API changes. But we can't do that because changing a major version in golang is too difficult.
Also note, that we have two kinds of clients: cmd-line clients and go package clients, and changes that may break one type of client may not break the other one.
To summarize, we are incapable of doing proper version management until we separate the go package from the cmd-line tool.